internal/core/services/operation_manager/operation_manager.go

```diff
@@ -271,3 +273,61 @@ func (om *OperationManager) cancelOperation(ctx context.Context, schedulerName,
 
 	return nil
 }
+
+func (om *OperationManager) GrantLease(ctx context.Context, operation *operation.Operation, initialTTL time.Duration) error {
+	err := om.leaseStorage.GrantLease(ctx, operation.SchedulerName, operation.ID, initialTTL)
+	if err != nil {
+		return fmt.Errorf("failed to grant lease to operation: %w", err)
+	}
+
+	return nil
+}
+
+func (om *OperationManager) RevokeLease(ctx context.Context, operation *operation.Operation) error {
+	err := om.leaseStorage.RevokeLease(ctx, operation.SchedulerName, operation.ID)
+	if err != nil {
+		return fmt.Errorf("failed to revoke lease to operation: %w", err)
+	}
+
+	return nil
+}
+
+func (om *OperationManager) StartLeaseRenewGoRoutine(operationCtx context.Context, op *operation.Operation, ttl time.Duration) {
```

Comment:
@caiordjesus currently we are receiving the ttl in both renew lease and grant lease methods as an argument. This means that the worker will have the responsibility of knowing the value it will pass to those methods. I was thinking that the best approach for setting this value would be isolating it into a config level parameter (same as the room initialization timeout on `internal/core/services/room_manager/room_manager.go:108`) instead of creating a constant value or just a hardcoded one.  With this in view, it would be necessary to include a new dependency of a config property similar to what we do in `RoomManager`, but with this ttl value being passed as an argument this config dependency would be injected in the `worker` and not the `operation manager`. Since we are already defined that the operation manager being responsible for encapsulating this kind of logic, what do you think about having it in there (doesn't need to be in this PR)?